nipype/interfaces/niftyreg/reg.py

```diff
@@ -0,0 +1,439 @@
+                      argstr='-rmask %s')
+    # Input floating mask
+    fmask_file = File(exists=True,
+                      desc='The input floating mask',
+                      argstr='-fmask %s')
+    # Maximum number of iterations
+    maxit_val = PositiveInt(desc='Maximum number of iterations',
+                            argstr='-maxit %d')
+    # Multiresolution levels
+    ln_val = PositiveInt(desc='Number of resolution levels to create',
+                         argstr='-ln %d')
+    # Number of resolution levels to process
+    lp_val = PositiveInt(desc='Number of resolution levels to perform',
+                         argstr='-lp %d')
+    # Smoothing to apply on reference image
+    desc = 'Amount of smoothing to apply to reference image'
+    smoo_r_val = traits.Float(desc=desc,
+                              argstr='-smooR %f')
+    # Smoothing to apply on floating image
+    desc = 'Amount of smoothing to apply to floating image'
+    smoo_f_val = traits.Float(desc=desc,
+                              argstr='-smooF %f')
+    # Use nifti header to initialise transformation
+    desc = 'Use nifti header to initialise transformation'
+    nac_flag = traits.Bool(desc=desc,
+                           argstr='-nac')
+    # Use the input masks centre of mass to initialise the transformation
+    desc = 'Use the masks centre of mass to initialise the transformation'
+    cog_flag = traits.Bool(desc=desc,
+                           argstr='-cog')
+    # Percent of blocks that are considered active.
+    v_val = PositiveInt(desc='Percent of blocks that are active',
+                        argstr='-pv %d')
+    # Percent of inlier blocks
+    i_val = PositiveInt(desc='Percent of inlier blocks', argstr='-pi %d')
+    # Lower threshold on reference image
+    ref_low_val = traits.Float(desc='Lower threshold value on reference image',
+                               argstr='-refLowThr %f')
+    # Upper threshold on reference image
+    ref_up_val = traits.Float(desc='Upper threshold value on reference image',
+                              argstr='-refUpThr %f')
+    # Lower threshold on floating image
+    flo_low_val = traits.Float(desc='Lower threshold value on floating image',
+                               argstr='-floLowThr %f')
+    # Upper threshold on floating image
+    flo_up_val = traits.Float(desc='Upper threshold value on floating image',
+                              argstr='-floUpThr %f')
+    # Platform to use
+    platform_val = traits.Int(desc='Platform index',
+                              argstr='-platf %i')
+    # Platform to use
+    gpuid_val = traits.Int(desc='Device to use id',
+                           argstr='-gpuid %i')
+    # Verbosity off
+    verbosity_off_flag = traits.Bool(argstr='-voff',
+                                     desc='Turn off verbose output')
+    # Set the number of omp thread to use
+    omp_core_val = traits.Int(desc='Number of openmp thread to use',
```

Comment:
Yes. If the omp_core_val is set, it is used, otherwise the environment variable. If none are set, the maximum number of thread is used, as defined by omp_get_num_procs() in C.